releasenotes/notes/annotated-params-116288d5628f7ee8.yaml

```
@@ -0,0 +1,23 @@
+---
+features_circuits:
+  - |
+    Add support for :meth:`.AnnotatedOperation.params`, which enables
+    circuit-level parameter handling (such as binding parameters) for
+    annotated operations.
+fixes:
+  - |
+    Fix a series of issues when controlling parameterized standard gates.
```

This is optional but I think it makes the reno nicer :)

```suggestion
    Fixed a series of issues when controlling parameterized standard gates.
```